source/whats-new.txt

```diff
@@ -37,6 +37,11 @@ What's New in 4.9
   ``BsonProperty`` or ``BsonId``.
 - Changed the log message format for command monitoring messages reported
   by the ``org.mongodb.driver.protocol.command`` package.
+- Added support for the ``$documents`` aggregation pipeline stage to the 
+  ``Aggregates`` helper class.
+- Added ``MongoClientSettings.applyToLoggerSettings()`` and 
+  ``LoggerSettings.Builder.maxDocumentLength()`` methods that let you specify a 
+  maximum length for the extended JSON representation of a BSON document in a log message. 
```

Comment:
Suggestion:
Since these are new methods and we don't have documentation that covers them, I wonder if it might be helpful to link to the API documentation for them.